js/repl/CodeMirror.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 
 import { css } from 'glamor';
+import CodeMirror from 'codemirror'
```

Comment:
I chose to go with CDN-based `CodeMirror` so we could share with Prettier and others.